server/services/AlertService.js

```diff
@@ -75,7 +75,8 @@ export default class AlertService {
     params.alertState = alertState;
     params.searchString = search;
     if (search.trim()) params.searchString = `*${search.trim().split(' ').join('* *')}*`;
-    if (monitorIds.length > 0) params.monitorId = monitorIds[0];
+    if (monitorIds.length > 0)
+      params.monitorId = !Array.isArray(monitorIds) ? monitorIds : monitorIds[0];
```

Comment:
Since `monitorIds` is defaulted to `[]` would it be possible to just make this:

```
params.monitorId = monitorIds.length > 0 ? monitorIds[0] : monitorIds
```

Or can there be a situation where the set value isn't an `Array`?

Reply:
Yea, there can be a situation where the value is not an `Array`. I found this out, so that is why I am making the bug fix.

Reply:
Gotcha, so when it isn't an `Array` is it a `String`? Just want to make sure `monitorIds.length` is a valid property in those scenarios

Reply:
yea that is right